### PR TITLE
Sidebar layout support for the user portal

### PR DESCRIPTION
**Pull request by brionmario**
Screen Shot 2019-09-04 at 4 07 21 AM
Screen Shot 2019-09-04 at 4 08 33 AM
Screen Shot 2019-09-04 at 4 08 44 AM

### CODE REVIEW

File: modules/theme/src/index.js [hunk 1]
```diff
@@ -31,3 +31,11 @@ export const HomeTileIcons = {
     security: require("../lib/assets/images/home_security_mgt.png"),
     consent: require("../lib/assets/images/home_consent_mgt.png")
 };
+export const SidePanelIconSet = {
```
Comment (jeradrutnam) [inline modules/theme/src/index.js]: Move icons to a folder

File: modules/theme/src/themes/default/assets/images/controls-icon.svg [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<svg version="1.1" class="icon" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/assets/images/controls-icon.svg]: Fix `id`. it has to be icon name

File: modules/theme/src/themes/default/assets/images/dashboard-icon.svg [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<svg version="1.1" class="icon" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```
Comment (jeradrutnam) [inline modules/theme/src/themes/default/assets/images/dashboard-icon.svg]: Fix `id`. it has to be icon name

Comment (brionmario) [reply modules/theme/src/themes/default/assets/images/dashboard-icon.svg]: Added the icon name as the id.

File: modules/theme/src/themes/default/assets/images/controls-icon.svg (hunk 2 above)
Comment (brionmario) [reply modules/theme/src/themes/default/assets/images/controls-icon.svg]: Added the icon name as the id.

File: modules/theme/src/index.js (hunk 1 above)
Comment (brionmario) [reply modules/theme/src/index.js]: Moved icons to a separate folder called `icons`

**Pull request merged**